src/xs_driver.cpp

```diff
@@ -714,11 +714,11 @@ bool InterbotixDriverXS::get_joint_states(
     if (positions) {
       if (is_motor_gripper(joint_name)) {
         positions->push_back(
-          robot_positions.at(get_js_index(joint_name)) - \
-          gripper_map[joint_name].calibration_offset);
+          robot_positions.at(get_js_index(joint_name)));
+          // gripper_map[joint_name].calibration_offset);
```

Comment:
Commented line?

Reply:
Yes we do not need the offset anymore. Kept this commented so that we know in future if we change the order, I can remove it if we do not like it there.

Reply:
Yes, please remove the commented line.